openprescribing/measures/definitions/nhse_bgts.json

```
@@ -40,19 +40,23 @@
     "~0601060D0EJAAA0 # True Metrix testing strips (Trividia Health UK Ltd)",
     "~0601060D0CIAAA0 # WaveSense JAZZ testing strips (AgaMatrix Europe Ltd)",
     "~0601060D0CIABA0 # WaveSense JAZZ Duo testing strips (AgaMatrix Europe Ltd)",
-    "~0601060D0CUABA0 # GlucoRx Nexus testing strips (GlucoRx Ltd)",
     "~0601060D0FCAAA0 # CareSens S testing strips (Spirit Healthcare Ltd)",
-    "~0601060D0FDAAA0 # AgaMatrix Agile Test Strips (AgaMatrix Europe Ltd)"
+    "~0601060D0FDAAA0 # AgaMatrix Agile Test Strips (AgaMatrix Europe Ltd)",
+    "~0601060D0CUAGA0 # GlucoRx Vivid testing strips (GlucoRx Ltd)",
+    "~0601060D0EFAAA0 # palmdoc testing strips (Palmdoc Ltd)",
+    "#Neon GK+ Glucose testing strips (Neon Diagnostics Ltd) WAITING FOR BNF CODE",
+    "#ALLYbg Test Strips WAITING FOR INCLUSION IN dm+d AND BNF CODE",
+    "GlucoRx Smart Glucose testing strips (GlucoRx Ltd) WAITING FOR BNF CODE"
```

```suggestion
    "# GlucoRx Smart Glucose testing strips (GlucoRx Ltd) WAITING FOR BNF CODE"
```
I think needs the # at the front to avoid causing an error?